src/udev/udevd.c

```diff
@@ -1759,6 +1759,9 @@ int main(int argc, char *argv[]) {
 
         r = run(fd_ctrl, fd_uevent, cgroup);
 
+        fd_ctrl = -1;
+        fd_uevent = -1;
+
```

Comment:
If `run()` returns `< 0`, the fds are still valid, right? So this should be conditional.